conda_build/build.py

```diff
@@ -394,11 +394,11 @@ def create_env(prefix, specs, clear_cache=True, debug=False):
         cc.pkgs_dirs = cc.pkgs_dirs[:1]
         actions = plan.install_actions(prefix, index, specs)
         plan.display_actions(actions, index)
-        plan.execute_actions(actions, index, verbose=debug)
+
         if on_win:
-            os.environ = {k.encode(codec) if hasattr(k, 'encode') else k:
-                            v.encode(codec) if hasattr(v, 'encode') else v
-                            for k, v in os.environ.items()}
+            for k, v in os.environ.items():
+                os.environ[k] = str(v)
+        plan.execute_actions(actions, index, verbose=debug)
 
```

Comment:
Yes.  I'm pretty sure that any error originating from this issue happens in the execute_actions call.  Thus, it is essential that our stringification happens before then.